openapi/src/parts/folder.yaml

```diff
@@ -99,6 +99,52 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/workspaces/v1/{workspaceId}/folders/{folderId}/properties:
+    parameters:
+    - $ref: '#/components/parameters/WorkspaceId'
+    - $ref: '#/components/parameters/FolderId'
+    post:
+      operationId: updateFolderProperties
+      summary: |
+        Update folder properties. Only properties with keys in request are
```

Comment:
Do you mind updating this as well? It's not clear if all properties are overwritten, or just the ones in the request.

https://cs.github.com/DataBiosphere/terra-workspace-manager/blob/main/openapi/src/parts/workspace.yaml#L242